nft_ingester/src/program_transformers/token_metadata/v1_asset.rs

```diff
@@ -223,6 +223,8 @@ pub async fn save_v1_asset<T: ConnectionTrait + TransactionTrait>(
                     asset::Column::Leaf,
                     asset::Column::Compressed,
                     asset::Column::Compressible,
+                    asset::Column::DataHash,
+                    asset::Column::CreatorHash,
```

Comment:
Good catch on missed columns.

Nitpick: I would put these in same order as they are in the model above (hashes first then compressed/compressible) just for consistency.